tests/serverpod_test_server/test_integration/authentication/email_auth_provider_test.dart

```diff
@@ -386,4 +386,65 @@ void main() async {
       expect(actualPasswordHash, notHunter4PasswordHash);
     });
   });
+
+  group('Given create account request without hashing generator', () {
```

Comment:
Suggestion: Please also add a test validating that our password migration isn't triggered if any of these are set.

Reply:
For example, Given custom hash generator and a stored legacy password in the database when user is authenticating then password is not migrated.